<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>thresholdsetDialog</class>
 <widget class="QDialog" name="thresholdsetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>485</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>保护阈值设置</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>procvalue.png</normaloff>procvalue.png</iconset>
  </property>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>337</y>
     <width>231</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>功率模块电感过温动作值</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>294</y>
     <width>54</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>温度2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>212</y>
     <width>63</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>报警值</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>147</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>输出电压保护值</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>48</y>
     <width>54</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string> 下 限</string>
   </property>
  </widget>
  <widget class="mylineedit" name="scbhLowerEdit">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>49</y>
     <width>162</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>48</y>
     <width>54</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string> 上 限</string>
   </property>
  </widget>
  <widget class="mylineedit" name="scbhUpEdit">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>49</y>
     <width>162</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="scbhSetBtn">
   <property name="geometry">
    <rect>
     <x>516</x>
     <y>48</y>
     <width>75</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>conf.png</normaloff>conf.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>92</y>
     <width>315</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>功率模块输入直流电压过压保护值</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>130</y>
     <width>54</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string> 下 限</string>
   </property>
  </widget>
  <widget class="mylineedit" name="gybhLowerEdit">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>131</y>
     <width>162</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>130</y>
     <width>54</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string> 上 限</string>
   </property>
  </widget>
  <widget class="mylineedit" name="gybhUpEdit">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>131</y>
     <width>162</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="gybhSetBtn">
   <property name="geometry">
    <rect>
     <x>516</x>
     <y>130</y>
     <width>75</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
   <property name="icon">
    <iconset>
     <disabledoff>conf.png</disabledoff>
    </iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>174</y>
     <width>231</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>功率模块输出电流值上限</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>212</y>
     <width>63</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保护值</string>
   </property>
  </widget>
  <widget class="mylineedit" name="scdlProcEdit">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>213</y>
     <width>162</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="mylineedit" name="scdlWarningEdit">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>213</y>
     <width>162</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="scdlSetBtn">
   <property name="geometry">
    <rect>
     <x>516</x>
     <y>212</y>
     <width>75</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
   <property name="icon">
    <iconset>
     <disabledoff>conf.png</disabledoff>
    </iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>256</y>
     <width>235</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>功率模块IGBT过温动作值</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>294</y>
     <width>54</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>温度1</string>
   </property>
  </widget>
  <widget class="mylineedit" name="IGBTgw1Edit">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>295</y>
     <width>162</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="mylineedit" name="IGBTgw2Edit">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>295</y>
     <width>162</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="IGBTgwSetBtn">
   <property name="geometry">
    <rect>
     <x>516</x>
     <y>294</y>
     <width>75</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
   <property name="icon">
    <iconset>
     <disabledoff>conf.png</disabledoff>
    </iconset>
   </property>
  </widget>
  <widget class="mylineedit" name="dggw1Edit">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>377</y>
     <width>162</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>340</y>
     <width>231</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string> 总电流平均值</string>
   </property>
  </widget>
  <widget class="mylineedit" name="dggw2Edit">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>377</y>
     <width>162</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="dggwSetBtn">
   <property name="geometry">
    <rect>
     <x>516</x>
     <y>376</y>
     <width>75</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
   <property name="icon">
    <iconset>
     <disabledoff>conf.png</disabledoff>
    </iconset>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mylineedit</class>
   <extends>QLineEdit</extends>
   <header>mylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
